src/org/thoughtcrime/securesms/preferences/AdvancedPreferenceFragment.java

```diff
@@ -65,21 +61,11 @@ public void onCreate(Bundle paramBundle) {
     preferE2eeCheckbox = (CheckBoxPreference) this.findPreference("pref_prefer_e2ee");
     preferE2eeCheckbox.setOnPreferenceChangeListener(new PreferE2eeListener());
 
-    inboxWatchCheckbox = (CheckBoxPreference) this.findPreference("pref_inbox_watch");
-    inboxWatchCheckbox.setOnPreferenceChangeListener((preference, newValue) ->
-      handleImapCheck(preference, newValue, CONFIG_INBOX_WATCH)
-    );
-
     sentboxWatchCheckbox = (CheckBoxPreference) this.findPreference("pref_sentbox_watch");
     sentboxWatchCheckbox.setOnPreferenceChangeListener((preference, newValue) ->
       handleImapCheck(preference, newValue, CONFIG_SENTBOX_WATCH)
```

Comment:
I think this check is not needed, it warns me whenever I try to disable watching `Sent` that it is somehow dangerous and I should make sure it's the same on other devices etc. etc.

Reply:
done